docs/js/main.js

```diff
@@ -144,112 +197,429 @@ function versionCompare(base, target) { // Return -1, 0, 1
-          document.getElementById("no-matches").remove();
-      }
-  
-      count = 0; // reset
-    }, 100); // Spam delay for better performance
+  searchResultBox = document.getElementById("search-bar-after");
+  if (count > 0 && (version != "" || searchValue != "" || filterType || filterNew)) {
+    searchResultBox.textContent = `${count} ${resultsFoundText}`
+    searchResultBox.style.display = null;
+  } else {
+    searchResultBox.style.display = "none";
+  }
 
-  }); 
+  if (count == 0) {
+    if (document.getElementById("no-matches") == null)
+    document.getElementById("content").insertAdjacentHTML('beforeend', '<p id="no-matches" style="text-align: center;">No matches found.</p>');
+  } else {
+    if (document.getElementById("no-matches") != null)
+      document.getElementById("no-matches").remove();
+  }
+
+  count = 0; // reset
 }
-// Search Bar </>
 
-// <> HighlightJS 
-document.querySelectorAll('pre.code').forEach(el => { // Apply the code formatting on the same <pre> not the <code> inside to not break the styling
-  hljs.highlightElement(el);
-});
-document.querySelectorAll('div .skript-code-block').forEach(el => { // This lags the docs pages due to the huge amount of elements being parsed, we can disable this if lag is so bad for some people or keep it because it looks AMAZING!
-  hljs.highlightElement(el);
-});
-// HighlightJS </>
+if (searchBar) {
+  searchBar.focus() // To easily search after page loading without the need to click
+  searchBar.addEventListener('keydown', (event) => {
+    setTimeout(() => { // Important to actually get the value after typing or deleting + better performance
+      searchNow();
+    }, 100);
+  });
+}
+// Search Bar </>
 
 // <> Placeholders
-function replacePlaceholders(html) {
-  let innerHTML = html.innerHTML;
+const ghAPI = "https://api.github.com/repos/SkriptLang/Skript"
+
+function replacePlaceholders(element) {
+  let innerHTML = element.innerHTML;
   if (innerHTML.includes("${latest-version}")) {
-    let lv = $.getJSON("https://api.github.com/repos/SkriptLang/Skript/releases?per_page=1", (data) => {
-      html.innerHTML = html.innerHTML.replaceAll("${latest-version}", data[0]["tag_name"]);
-    })
+    getApiValue(element, "latest-version", "releases", (data) => {
+      return data[0]["tag_name"];
+    });
+  }
+
+  if (innerHTML.includes("${latest-version-changelog}")) {
+    getApiValue(element, "latest-version-changelog", "releases", (data) => {
+      return data["body"].replaceAll("\\r\\n", "<br>");
+    });
+  }
+
+  if (innerHTML.includes("${stable-version}")) {
+    getApiValue(element, "stable-version", "releases/latest", (data) => {
+      return data["tag_name"];
+    });
+  }
+
+  if (innerHTML.includes("${stable-version-changelog}")) {
+    getApiValue(element, "stable-version-changelog", "releases/latest", (data) => {
+      return data["body"].replaceAll("\\r\\n", "<br>");
+    });
+  }
+
+  if (innerHTML.includes("${latest-issue-")) {
```

Comment:
Where are these variables used?

Reply:
It's for the future usage, currently no where